src/main/resources/templates/email/organization/missingContact/body.ftlh.mjml

```diff
@@ -9,13 +8,11 @@
         <mj-section padding="0px">
             <mj-column mj-class="body-wrapper">
                 <mj-text mj-class="text-headline06-bold">
-                ${strings("notification.observation.notScheduledSupport.email.body.1", organizationName, plantingSiteName)}
+                    Organization ${organizationName} has no contact!
```

Comment:
Is the assumption that the "support mailing list" recipients will always speak English? Is someone at Terraformation in the support mailing list?

Reply:
Yes, the support mailing list is an internal thing and it will always be English-speaking TF employees reading it.